aptos-move/framework/aptos-framework/sources/types.move

```diff
@@ -0,0 +1,42 @@
+/// Common types.
+module aptos_framework::types {
```

Comment:
the name seems too generic, why not just validator_consensus_info? then each function name can be shorter too

Reply:
fixed [here](https://github.com/aptos-labs/aptos-core/pull/11741/files#diff-e182e8252d937c6257713cc0784a87bce27801f4e23837a376214bc443d3b0d5)